user/chrome.md

```diff
@@ -19,3 +20,40 @@ addons:
 ## Headless mode
 
 You can use Google Chrome in [headless mode](/user/gui-and-headless-browsers/#Using-the-Chrome-addon-in-the-headless-mode).
+
+## Sandboxing
+
+For security reasons, Google Chrome is unable to provide sandboxing when it is running in the
+[container-based environment](https://docs.travis-ci.com/user/reference/overview/#Virtualization-environments).
+
+To use Chrome in the container-based environment:
+
+1. Pass `--no-sandbox` when invoking the `chrome` command.
+
+  ```yaml
+  sudo: false
+  addons:
+    chrome: stable
+  script:
+    - chrome --no-sandbox
+  ```
+{: data-file=".travis.yml"}
+1. Your testing framework may have a different mechanism to supply this flag to the `chrome` executable.
+  For example, with [the customlauncher plugin for Karma](https://github.com/karma-runner/karma-chrome-launcher), you would add it to the `flags` array:
+  
+  ```javascript
+  module.exports = function(config) {
+    config.set({
+      browsers: ['Chrome', 'ChromeHeadless', 'ChromeHeadlessNoSandbox'],
+
+      // you can define custom flags
+      customLaunchers: {
+        Chrome_no_sandbox: {
```

Comment:
For this to be a valid configuration, the key name must match what is in the above `browsers` array, such as `MyHeadlessChrome` in this example: https://github.com/ebidel/headless-karma-travis/blob/da2131fbe130c45d32ea3fa138d9d475baf81f7e/karma.conf.js#L12-L18 (IIUC)